packages/wrapped-keys/esbuild.config.js

```diff
@@ -0,0 +1,28 @@
+const esbuild = require('esbuild');
+
+(async () => {
+  await esbuild.build({
+    entryPoints: [
+      './src/lib/litActions/solana/src/signTransactionWithSolanaEncryptedKey.js',
+      './src/lib/litActions/solana/src/signMessageWithSolanaEncryptedKey.js',
+      './src/lib/litActions/solana/src/generateEncryptedSolanaPrivateKey.js',
+    ],
+    bundle: true,
+    minify: true,
+    sourcemap: false,
+    outdir: './src/lib/litActions/solana/dist',
+    inject: ['./buffer.shim.js'],
+  });
+  await esbuild.build({
+    entryPoints: [
+      './src/lib/litActions/ethereum/src/signTransactionWithEthereumEncryptedKey.js',
+      './src/lib/litActions/ethereum/src/signMessageWithEthereumEncryptedKey.js',
+      './src/lib/litActions/ethereum/src/generateEncryptedEthereumPrivateKey.js',
+    ],
+    bundle: true,
+    minify: true,
+    sourcemap: false,
+    outdir: './src/lib/litActions/ethereum/dist',
+    inject: ['./buffer.shim.js'],
+  });
```

Comment:
Not sure when it was added, but I don't think this needs to be here twice

Reply:
We publishing Ethereum Lit Actions to IPFS just like Solana

Reply:
@spacesailor24 If you look at the files in the `entryPoints` they are actually different files. Solana & Ethereum